src/main/kotlin/processing/filters/RGBIntensity.kt

```diff
@@ -29,4 +29,6 @@ class RGBIntensity(private val factor: Double, private val type: RGBType) :
             }
         }
     }
+
+    override fun toString(): String = "${type} Intensity ${(factor * 100).toInt()}%"
```

Comment:
Just leave a comment here, when I try to print out RGB modification based on the version with 'adjust' button, nothing was displayed in the transform history list (displaying each RGBHSV value in adjust class result in empty string). so here might need some checking while merging.